gen/dcos-config.yaml

```diff
@@ -2772,9 +2772,9 @@ package:
           # HTTP Server
           # ===========
           # Enable/Disable the built-in HTTP Server for metrics
-          HTTP_Server  Off
-          #HTTP_Listen  0.0.0.0
-          #HTTP_Port    62020
+          HTTP_Server  On
+          HTTP_Listen  $DCOS_NODE_PRIVATE_IP
```

Comment:
`dcos-fluent-bit.service` isn't starting because this env var isn't defined, which is causing it to get passed literally as `$DCOS_NODE_PRIVATE_IP`.